edweiss-app/constants/Colors.ts

```diff
@@ -34,11 +34,12 @@ const Colors = {
 		surface0: "#ccd0da",
 		base: "#eff1f5",
 		mantle: "#e6e9ef",
-		crust: "#dce0e8"
+		crust: "#dce0e8",
+		Teal: "#8bd5ca",
 	},
 	dark: {
 		transparent: '#0000',
-
+		title: "#000000",
```

Comment:
It was precisely for all the course titles placed in the calendar, which contain colors. This allows them to be seen on the colors that remain the same in both themes. This does not change the overall theme. I will specify the name to avoid any confusion. 

`		course_title_for_backgroud_color: "#000000",`